2016/05/20 09:17:17 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:17:19 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:17:25 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:18:37 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "UnduhTransaksiPengeluaran". in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('UnduhTransaksiP...')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/UnduhTransaksiPengeluaran
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran
---
2016/05/20 09:19:33 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\LaporanPemasukkan.php:18
Stack trace:
#0 [internal function]: YiiBase::autoload('LaporanPemasukk...')
#1 C:\xampp\htdocs\siadm\protected\controllers\UnduhController.php(92): spl_autoload_call('LaporanPemasukk...')
#2 [internal function]: UnduhController->actionExcelPemasukkan('2016-05-01', '2016-05-20')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UnduhController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UnduhController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('excelPemasukkan')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('unduh/excelPema...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/unduh/excelPemasukkan?awal=2016-05-01&akhir=2016-05-20
HTTP_REFERER=http://localhost/siadm/index.php/Unduh/UnduhTransaksiPengeluaran
---
2016/05/20 09:22:08 [error] [exception.CException] CException: Property "TransaksiPengeluaran.ID" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('ID')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('ID')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:23:14 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:23:17 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:24:14 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:25:28 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:28:14 [error] [exception.CException] CException: Class name "Log" does not match class file "C:\xampp\htdocs\siadm\protected\models\Log.php". in C:\xampp\htdocs\yii\YiiBase.php:423
Stack trace:
#0 [internal function]: YiiBase::autoload('Log')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(14): spl_autoload_call('Log')
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create Transaks...')
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#6 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#7 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#17 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#19 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:28:59 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'siadmdb.log' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `log`.
in C:\xampp\htdocs\siadm\protected\models\Base.php (14)
in C:\xampp\htdocs\siadm\protected\models\Base.php (29)
in C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php (75)
2016/05/20 09:28:59 [error] [exception.CDbException] CDbException: The table "log" for active record class "Log" cannot be found in the database. in C:\xampp\htdocs\yii\db\ar\CActiveRecord.php:2387
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Log))
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(79): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(14): CActiveRecord->__construct()
#3 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create Transaks...')
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#5 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#6 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#7 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#8 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#18 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#20 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 04:21:09 [error] [exception.CException] exception 'CException' with message 'Property "Santri.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\SantriController.php(95): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SantriController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('Santri/create')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/Santri/create
HTTP_REFERER=http://localhost/siadm/index.php/Santri/create
---
2016/05/20 04:27:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "penilaian".' in C:\xampp\htdocs\yii\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(270): CController->missingAction('penilaian')
#1 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('penilaian')
#2 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/penilaian')
#3 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#5 {main}
REQUEST_URI=/siadm/index.php/site/penilaian
---
2016/05/20 04:54:04 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(UserWeb), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('donatur')
#9 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#11 {main}
REQUEST_URI=/siadm/index.php/donatur
---
2016/05/20 04:54:45 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(UserWeb), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#9 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#11 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran
---
2016/05/20 04:55:55 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(UserWeb), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#9 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#11 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
---
2016/05/20 05:07:54 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/1?id2=2&id3=2
---
2016/05/20 05:08:41 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
---
2016/05/20 05:08:43 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
---
2016/05/20 05:08:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
---
2016/05/20 05:09:24 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update
---
2016/05/20 05:09:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/vi...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/view
---
2016/05/20 05:09:41 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
---
2016/05/20 05:10:26 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/vi...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/view
---
2016/05/20 05:10:27 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update
---
2016/05/20 05:10:28 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
---
2016/05/20 05:10:33 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/1
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/1?id2=2&id3=2
---
2016/05/20 05:11:22 [error] [exception.CException] exception 'CException' with message 'Property "alokasiKelas.id2" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id2')
#1 C:\xampp\htdocs\siadm\protected\controllers\alokasiKelasController.php(117): CActiveRecord->__get('id2')
#2 [internal function]: alokasiKelasController->actionUpdate('2', '2', '2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(alokasiKelasController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(alokasiKelasController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
---
2016/05/20 05:11:36 [error] [exception.CException] exception 'CException' with message 'Property "alokasiKelas.id2" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id2')
#1 C:\xampp\htdocs\siadm\protected\controllers\alokasiKelasController.php(117): CActiveRecord->__get('id2')
#2 [internal function]: alokasiKelasController->actionUpdate('2', '2', '2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(alokasiKelasController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(alokasiKelasController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
---
2016/05/20 05:11:49 [error] [exception.CException] exception 'CException' with message 'Property "alokasiKelas.id2" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id2')
#1 C:\xampp\htdocs\siadm\protected\controllers\alokasiKelasController.php(117): CActiveRecord->__get('id2')
#2 [internal function]: alokasiKelasController->actionUpdate('2', '2', '2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(alokasiKelasController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(alokasiKelasController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
---
2016/05/20 05:13:49 [error] [exception.CException] exception 'CException' with message 'Property "alokasiKelas.id2" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id2')
#1 C:\xampp\htdocs\siadm\protected\controllers\alokasiKelasController.php(117): CActiveRecord->__get('id2')
#2 [internal function]: alokasiKelasController->actionUpdate('2', '2', '2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(alokasiKelasController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(alokasiKelasController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('update')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('alokasikelas/up...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
HTTP_REFERER=http://localhost/siadm/index.php/alokasikelas/update/2?id2=2&id3=2
---
2016/05/20 05:24:16 [error] [exception.CException] exception 'CException' with message 'Property "TransaksiPengeluaran.Deskripsi" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('Deskripsi')
#1 C:\xampp\htdocs\siadm\protected\models\TransaksiPengeluaran.php(100): CActiveRecord->__get('Deskripsi')
#2 C:\xampp\htdocs\siadm\protected\views\transaksiPengeluaran\index.php(20): TransaksiPengeluaran->search()
#3 C:\xampp\htdocs\yii\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\yii\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\web\CController.php(782): CController->renderPartial('index', Array, true)
#7 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(140): CController->render('index', Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionIndex()
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran
HTTP_REFERER=http://localhost/siadm/index.php/site/index
---
2016/05/20 05:26:26 [error] [exception.CException] exception 'CException' with message 'Property "TransaksiPengeluaran.tanggal_pengeluaran" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('tanggal_pengelu...')
#1 C:\xampp\htdocs\siadm\protected\models\TransaksiPengeluaran.php(101): CActiveRecord->__get('tanggal_pengelu...')
#2 C:\xampp\htdocs\siadm\protected\views\transaksiPengeluaran\index.php(20): TransaksiPengeluaran->search()
#3 C:\xampp\htdocs\yii\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\yii\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\web\CController.php(782): CController->renderPartial('index', Array, true)
#7 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(140): CController->render('index', Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionIndex()
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran
HTTP_REFERER=http://localhost/siadm/index.php/site/index
---
